modules/util/portal-tools-db-upgrade-client/src/testFunctional/ant/build-test-db-upgrade-client.xml

```diff
@@ -299,6 +299,41 @@
 		<clean-database-upgrade-client />
 	</target>
 
+	<target name="check-upgrade-client-home-properties">
+		<get-database-upgrade-client-paths />
+
+		<prepare-database-upgrade-configuration />
+
+		<property name="user.home" value="~" />
+
+		<execute dir="${user.home}" failonerror="true">
+			touch portal-ext.properties
+		</execute>
+
+		<execute dir="${db.upgrade.client.home}" failonerror="true">
+			db_upgrade${file.suffix.bat} run > temp.log
+		</execute>
+
+		<loadfile
+			property="upgrade.process.output"
+			srcFile="temp.log"
+		/>
+
+		<echo>${upgrade.process.output}</echo>
+
+		<if>
+			<contains string="${upgrade.process.output}" substring="Remove /home/sofiamendoza/portal-ext.properties prior to running an upgrade to prevent possible conflicts." />
```

Comment:
I think we should modify the substring to "portal-ext.properties prior to running an upgrade to prevent possible conflicts." because CI's home user is different

Reply:
Or "Remove /${USER}/portal-ext.properties prior to running an upgrade to prevent possible conflicts." would be better if you can find the user

Reply:
She fixes that at : https://github.com/lucasperj/liferay-portal/pull/311/commits/c2c8153fa0fd78c93edae6a0ab82da2547254658

Reply:
`/home/${user.name}/portal-ext.properties` can work?

Reply:
o woops didnt see that

Reply:
ignore these requested changes